setup.py

```diff
@@ -2,17 +2,18 @@
 
 setup(
     name='drf-oidc-auth',
-    version='0.10.0',
+    version='0.11.0',
     packages=['oidc_auth'],
     url='https://github.com/ByteInternet/drf-oidc-auth',
     license='MIT',
     author='Maarten van Schaik',
     author_email='support@byte.nl',
     description='OpenID Connect authentication for Django Rest Framework',
     install_requires=[
-        'pyjwkest>=1.0.3',
+        'authlib>=0.15.0',
```

Comment:
Perhaps we have to add a certain version of cryptography in here? It seems that the `authlib` change requires us the have a specific version of cryptography. I.e. it refers to this: https://github.com/lepture/authlib/blob/master/authlib/jose/rfc8037/okp_key.py#L1 which does not exist in older versions of cryptography.

Reply:
Ahh right.
Could you send me the version here so I could at it accordingly?.

Reply:
Yeah, looks like it stops complaining if you upgrade to version 2.6 at least. :)